incubator/java-openliberty/templates/binary/pom.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version='1.0' encoding='utf-8'?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <!--required parent POM-->
+        <groupId>dev.appsody</groupId>
+        <artifactId>java-openliberty</artifactId>
+        <version>{{.stack.parentpomrange}}</version>
+    </parent>
+
+    <groupId>dev.appsody.starter.java-openliberty</groupId>
+    <artifactId>tamigration</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
```

Comment:
Not sure what type of packaging.. but since "war" is requiring us to disable the war plugin...  and since "jar" is more the default...maybe "jar" packaging?     

One could also argue for "pom" packaging.  But it's not crazy to think someone could want to package a jar with all this.  

OTOH, it is a bit odd to have one stack containing multiple packaging types.  But "binary" is going to be a slightly odd-fit no matter what.   I still vote for .jar

Reply:
ok, updated to jar packaging